src/Components/Authentication/Types.ts

```diff
@@ -61,11 +56,6 @@ export interface ModalOptions {
    * Hook to be called after the modal has closed
    */
   afterClose?: () => void
-  /**
-   * MOBILE ONLY
-   * Used to construct afterSignupAction from query params
-   */
-  action?: AfterSignUpAction["action"]
```

Comment:
This was previously used by martsy to communicate with this 'new' component. I do not know why it wasn't nested in the first place. It no longer does anything.

Reply:
Ditto for the rest of these `MOBILE ONLY` props